### PR TITLE
test `ctc-address` `own-addr`

### PR DESCRIPTION
**Pull request by chikeabuah**

### CODE REVIEW

File: examples/simulator-own-addr/client.mjs [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const c = await import('@reach-sh/simulator-client');
```
Comment (jeapostrophe) [inline examples/simulator-own-addr/client.mjs:1]: I haven't noticed if the others are like this, but you should use the real import syntax

File: examples/simulator-own-addr/index.rsh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+'reach 0.1';
+'use strict';
+
+export const main =
+  Reach.App(
```
Comment (jeapostrophe) [inline examples/simulator-own-addr/index.rsh:5]: You should not commit new code with the ternary syntax

Comment (jeapostrophe) [reply examples/simulator-own-addr/index.rsh:5]: Or is this just a copy of something else? If so, you could do the thing where you copy and example into this directory.

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

File: examples/simulator-own-addr/client.mjs (hunk 1 above)
Comment (chikeabuah) [reply examples/simulator-own-addr/client.mjs:1]: @jeapostrophe what's wrong with this syntax? i prefer this syntax because it also works in the node REPL

Comment (chikeabuah) [reply examples/simulator-own-addr/client.mjs:1]: every simulator example is written like this

Comment (chikeabuah) [reply examples/simulator-own-addr/client.mjs:1]: but okay, i'll change it if it's best practice, one moment

**Review by jeapostrophe — APPROVED** [review]

**Pull request merged**